generate_rospkg_apkbuild/genapkbuild.py

```diff
@@ -204,7 +204,7 @@ def package_to_apkbuild(ros_distro, package_name,
             f = open(pkglist, 'w')
             f.write(yaml.dump(rosinstall))
             f.close()
-            subprocess.check_output(['wstool', 'init', tmpd, pkglist])
+            subprocess.check_output(['vcs', 'import', '<', pkglist])
```

Comment:
I think `<` will be passed to `vcs` as an argument instead of treating it as a shell redirection.